cluster/gce/gci/health-monitor.sh

```diff
@@ -28,6 +28,11 @@ function docker_monitoring {
   while [ 1 ]; do
     if ! timeout 60 docker ps > /dev/null; then
       echo "Docker daemon failed!"
+      # Dump stack of docker daemon for investigation.
+      # Log fle name looks like goroutine-stacks-TIMESTAMP and will be saved to
+      # the exec root directory, which is /var/run/docker/ on Ubuntu and COS.
+      pkill -SIGUSR1 docker
```

Comment:
in order to take the dump, i send SIGUSR1 to dockerd or docker-containerd process. Not sure what `docker` would do  here. Would this also send the signal to the daemon `dockerd` ?

Reply:
Thanks @krmayankk. Naively, I tried to be consistent with the following line, but just realized that the pattern `docker` would match against `dockerd`, `docker-containerd`, and `docker-containerd-shim`. Will change to `dockerd`

Reply:
Changed to `dockerd`